### PR TITLE
Add cpu support

### PR DESCRIPTION
**Pull request by gc-fu**
This PR tries to add CPU support to Langchain-Chat-Chat

### CODE REVIEW

File: server/knowledge_base/kb_cache/base.py [hunk 1]
```diff
@@ -143,10 +143,14 @@ def load_embeddings(self, model: str = None, device: str = None) -> Embeddings:
                     else:
                         # maybe ReRanker or else, just use empty string instead
                         query_instruction = ""
-                    if device in ['xpu']:
+                    if device in ['xpu', 'cpu']:
                         from ipex_llm.langchain.embeddings import TransformersBgeEmbeddings
+                        low_bit_format = "fp16"
```
Comment (Oscilloscope98) [inline server/knowledge_base/kb_cache/base.py]: Maybe add a if/elseif logic here, or using switch?

Comment (gc-fu) [reply server/knowledge_base/kb_cache/base.py]: Fixed

**Comment by gc-fu** [comment]
README will be added in later PRs.

File: configs/server_config.py.example [hunk 2]
```diff
@@ -90,12 +90,12 @@ FSCHAT_MODEL_WORKERS = {
...
-    },
-    "Llama-2-7b-chat-hf": {
-        "device": "xpu",
-    },
+    # "chatglm3-6b": {
+    #     "device": "cpu",
+    # },
+    # "Llama-2-7b-chat-hf": {
+    #     "device": "cpu",
+    # },
```
Comment (Oscilloscope98) [inline configs/server_config.py.example]: Let's directly remove them :) ?

File: startup.py [hunk 3]
```diff
@@ -7,8 +7,10 @@
 from datetime import datetime
 from pprint import pprint
 from langchain_core._api import deprecated
+from configs.model_config import LLM_DEVICE
 
-import intel_extension_for_pytorch as ipex
+if LLM_DEVICE=="xpu":
```
Comment (Oscilloscope98) [inline startup.py]: Should we add a check here for LLM_DEVICE or EMBEDDING_DEVICE equal to xpu?

Comment (Oscilloscope98) [reply startup.py]: We should import ipex for the case that at least one model is on XPU (even we did not verify this case for now

**Review by Oscilloscope98 — APPROVED** [review]
LGTM if both work on CPU and GPU for now.

Let's add a new README in further PR

**Comment by gc-fu** [comment]
Have tested on both CPU and XPU.

**Pull request merged**